### PR TITLE
Cylance Protect v2 to package and handle 409 error gracefully

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17813
fixes: https://github.com/demisto/etc/issues/17809

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - no change
- [ ] Code Review

### CODE REVIEW

**Comment by bakatzir** [comment]
@yaakovi  changes due to our linters and mypy.
handle of 409 error in a separate commit.

File: Integrations/Cylance_Protect_v2/Cylance_Protect_v2_description.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (yaakovi) [inline Integrations/Cylance_Protect_v2/Cylance_Protect_v2_description.md]: if you don't have anything to write here, remove the file

File: Releases/LatestRelease/Cylance_Protect_v2.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Move integration code to package, handleerrors and logs more gracefully
```
Comment (yaakovi) [inline Releases/LatestRelease/Cylance_Protect_v2.md]:
```suggestion
handle errors and logs more gracefully.
```

Comment (yaakovi) [reply Releases/LatestRelease/Cylance_Protect_v2.md]: @kirbles19 
please review the release notes here.

**Comment by bakatzir** [comment]
@yaakovi do we want to force merge? Still fails on triggering other endpoint playbooks.

**Comment by yaakovi** [comment]
@bakatzir 
i'm waiting for @kirbles to review the yml

**Comment by bakatzir** [comment]
@yaakovi yml reviewed.

**Comment by bakatzir** [comment]
@yaakovi just reminding.

**Pull request merged**